faros_event.sh

```diff
@@ -346,37 +345,34 @@ function processEventTypes() {
 }
 
 function unix_millis_to_iso8601() {
-    echo $(jq -r '. / 1000 | todate' <<< $1)
+    jq -r '. / 1000 | todate' <<< "$1"
 }
 
 function make_commit_key() {
-    echo $(keys_matching "$flat" "data_commit_.*")
+    keys_matching "$flat" "data_commit_.*"
 }
 
 function make_artifact_key() {
     if ! [ -z "$has_artifact" ]; then
-        echo $(keys_matching "$flat" "data_artifact_.*")
+        keys_matching "$flat" "data_artifact_.*"
     else
-        echo $( jq -n \
-                    --arg commit_sha "$commit_sha" \
-                    --arg commit_repo "$commit_repo" \
-                    --arg commit_org "$commit_org" \
-                    --arg commit_source "$commit_source" \
-                    '{
-                        "data_artifact_id": $commit_sha,
-                        "data_artifact_repository": $commit_repo,
-                        "data_artifact_organization": $commit_org,
-                        "data_artifact_source": $commit_source,
-                    }'
-                )
+        jq -n \
+          --arg commit_sha "$commit_sha" \
+          --arg commit_repo "$commit_repo" \
+          --arg commit_org "$commit_org" \
+          --arg commit_source "$commit_source" \
+          '{
+              "data_artifact_id": $commit_sha,
+              "data_artifact_repository": $commit_repo,
+              "data_artifact_organization": $commit_org,
+              "data_artifact_source": $commit_source,
+          }'
     fi
 }
 
 function doCDMutations() {
     flat=$(flatten "$request_body")
 
-    deploy=$(keys_matching "$flat" "data_deploy_(id|environment|application|source)")
```

Comment:
this was unused - so I removed it

Reply:
@ypc-faros is this some sort of validation step?

Reply:
No, it's really unused